src/molecules/Table/index.ts

```diff
@@ -1 +1 @@
-export { default } from './Table';
+export { default as Table } from './Table';
```

Comment:
Sorry I meant this line should go in `src/modules/index.ts`, and this file should be left alone (like the previous commit)